src/main/browser/types.ts

```
@@ -1,42 +1,21 @@
 import { WebContentsView } from 'electron';
 
-import { ElementTarget, RecordedAction, TabInfo } from '@/shared/types';
-import { VideoRecorder } from '@/main/recording';
-import { BrowserAutomationExecutor } from '@/main/automation';
-import { PasswordAutomation } from '@/main/password';
+import { TabInfo } from '@/shared/types';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)